shell/platform/windows/external_texture_gl.cc

```diff
@@ -0,0 +1,126 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/external_texture_gl.h"
+
+// OpenGL ES and EGL includes
```

Comment:
This is obvious from the includes themselves; includes are generally only commented when there is something non-obvious that needs to be explained.